pynitrokey/cli/start.py

```diff
@@ -93,14 +90,85 @@ def rng(count, raw, quiet):
                 print(challenge.hex())
 
 
+class CardRemovedGPGAgentException(RuntimeWarning):
+    pass
+
+
+class ServiceNotRunningGPGAgentException(RuntimeWarning):
+    pass
+
+
+class NoCardPCSCException(RuntimeWarning):
+    pass
+
+
+def gpg_agent_set_identity(identity: int):
+    from subprocess import check_output as run
+
+    cmd = f"gpg-connect-agent" f"|SCD APDU 00 85 00 0{identity}" f"|/bye".split("|")
```

Comment:
You don't need an f-string for all of those strings.
Also, it might be worth providing a string list instead.